openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -452,7 +452,7 @@
                 // the staff override itself.
                 view.loadStudentInfo({expanded_view: 'final-grade'});
             };
-            this.callStaffAssess(submissionID, rubric, scope, successCallback, '.staff-override-error');
+            this.callStaffAssess(submissionID, rubric, scope, successCallback, '.staff-override-error', 'regrade');
```

Comment:
I don't see any unit testing in Python or JS for 'regrade' (the four occurrences in these changes are all in non-test code). It would be good to have a Jasmine test that verifies that the parameter is passed correctly to the server, and then to have a server test that verifies that the correct event is published.